test/DynamoCoreTests/DynamoCoreTests.csproj

```diff
@@ -37,16 +37,16 @@
         </PackageReference>
 
         <!--moq and deps - copied to test_dependencies folder-->
-        <PackageReference Include="Moq" Version="4.18.4" GeneratePathProperty="true" />
-        <PackageReference Include="Castle.Core" Version=" 5.1.1" GeneratePathProperty="true" />
+        <PackageReference Include="Moq" Version="4.18.4" ExcludeAssets="runtime" GeneratePathProperty="true" />
+        <PackageReference Include="Castle.Core" Version=" 5.1.1" ExcludeAssets="runtime" GeneratePathProperty="true" />
```

Comment:
What happens if you don't exclude these assets? Are they all copied to the test_dependencies path in that case?